Ziti.NET.Standard/src/OpenZiti/ZitiIdentity.cs

```diff
@@ -169,8 +169,12 @@ public class ZitiIdentity {
 			Native.ziti_options ziti_opts = new Native.ziti_options {
 				//app_ctx = GCHandle.Alloc(InitOpts.ApplicationContext, GCHandleType.Pinned),
 				config = InitOpts.IdentityFile,
-				config_types = cfgs,
-				refresh_interval = refreshInterval,
+				config_types = cfgs,
```

Comment:
all these architecture issues are sorted now, right? we can remove these changes?

Reply:
refresh interval has to be long in 64 bit processor

Reply:
the C Sharp sdk can provide an int here and not cast to long. If someone needs a refresh interval > 2billion seconds they can file a pull request :) 

You can remove the if/else and (long) cast, right? c# will automatically fit the incoming c# into into the long

Reply:
When I change the data type of refresh_interval to int, it is failing in MAC

Reply:
It'd be helpful to know how it fails. At runtime, at compile time, or something else? i'll see if i can replicate

Reply:
It is failing at runtime. I run this code using DEBUG | 64 or Release | 64 options in MAC.